spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,20 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'it locks the given account' do
+      other_user = FactoryGirl.create :user
```

Comment:
I used `user` over `other_user` in my example because I found the `other_` rather redundant, I didn't get what it would relate to. But I don't care too much either way.